element_electrode_localization/version.py

```diff
@@ -1,2 +1,2 @@
 """Package metadata."""
-__version__ = "0.1.5"
+__version__ = "0.1.6"
```

Comment:
```suggestion
__version__ = "0.2.0"
```
Backward-incompatible or major changes like these are the second number in SemVar.